src/yaml_to_warehouse.cpp

```diff
@@ -113,25 +106,36 @@ int main(int argc, char** argv)
     std::string scene = fmt::format("{}/scene{:04}.yaml", path, i);
     std::string scene_sensed = fmt::format("{}/scene_sensed{:04}.yaml", path, i);
     std::string request = fmt::format("{}/request{:04}.yaml", path, i);
-    std::string trajectory = fmt::format("{}/trajectory{:04}.yaml", path, i);
+    std::string trajectory = fmt::format("{}/path{:04}.yaml", path, i);
 
     if (yaml_msg::fromYAMLFile(scene_msg, scene))
     {
-      scene_msg.name = fmt::format("scene{}", i);
+      scene_msg.name = scene;
```

Comment:
Would you mind adding brief comments for each if and else scope that document what happens?